src/Microsoft.Tye.Hosting/Model/ServiceDescription.cs

```diff
@@ -25,13 +25,7 @@ public ServiceDescription(string name, RunInfo? runInfo)
 
     public class ConfigurationSource
     {
-        public ConfigurationSource(string name, string value)
-        {
-            Name = name;
-            Value = value;
-        }
-
-        public string Name { get; }
-        public string Value { get; }
+        public string? Name { get; set; }
```

Comment:
Organizationally, these are objects that are used by the JSON and non-JSON. I left them in place for now.